### ISSUE TITLE
How to send headers?

### ISSUE THREAD

**Issue by deb4it** [chrome-php/headless-chromium-php#149]
Hi @gsouf 

how can I send headers along with cookie and other data while fetching the webpage

Please give me your suggestions.

Many thanks!!!

**Comment by haris1024** [comment]
Have you read the project description (REAME.md)? there are examples of how to set headers and cookies.

**Comment by gsouf** [comment]
Has explained by @haris1024 it's described in the doc.

This is an example of how to attach cookies to a given page:

```
    $page->setCookies([
        Cookie::create('name', 'value', [
            'domain' => 'example.com',
            'expires' => time() + 3600 // expires in 1 day
        ])
    ])->await();
```

**Comment by CyberPunkCodes** [comment]
The use of additional headers is not documented in the readme. There are helper functions for setting a few things specifically, like cookies, user agent, etc.. but not for any additional headers.

I did manage to pass any extra headers I needed on the page instance. I was not able to get them on the initial connection on the browser factory.

My untested concern with setting it on the page, is if subsequent page requests lose the headers.

This is how I got it to work:

```php
$page = $browser->createPage();

$page->getSession()->sendMessageSync(new Message('Network.setExtraHTTPHeaders', [
    'headers' => [
        'Accept-Encoding' => 'identity',
        'Accept-Language' => 'en-US,en;q=0.9',
    ]
]));

$page->navigate($url);
```
